File: src/main/java/com/seedOrder/Maid_To_Order_Backend/repository/OrderRepository.java
package com.seedOrder.Maid_To_Order_Backend.repository;


import com.seedOrder.Maid_To_Order_Backend.model.OrderEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface OrderRepository extends JpaRepository<OrderEntity, Long> {
}

File: Maid-to-Order-Backend/src/main/kotlin/com/maidtoorder/backend/repository/OrderRepository.kt
package com.maidtoorder.backend.repository

import com.maidtoorder.backend.model.Order
import com.maidtoorder.backend.model.OrderStatus
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.time.LocalDateTime

@Repository
interface OrderRepository : JpaRepository<Order, Long> {
    fun findByStatus(status: OrderStatus): List<Order>
    fun findByCustomerPhone(phone: String): List<Order>
    fun findByCreatedAtBetween(start: LocalDateTime, end: LocalDateTime): List<Order>
}

